### PR TITLE
static string to integer map

### PR DESCRIPTION
**Pull request by RamanjaneyuluIdavalapati**
Ref: https://github.com/deep-compute/diskdict/pull/5

### CODE REVIEW

**Comment by RamanjaneyuluIdavalapati** [comment]
@jaswanth098 Can't able to add Reviewers and Assignees. Can you please check this.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # DiskDict
 
 ## Description
-`DiskDict` is a file based module which stores data as key, value pairs like a python dictionary.
+`DiskDict` is a file based module which stores data as key, value pairs like a python dictionary. `DiskDict` stores the data on the disk unlike python dictionary which stores data in the RAM.
 
 ## Installation
 
-> Prerequisites: Python
+> Prerequisites: Python3
```
Comment (prashanthellina) [inline README.md]: @RamanjaneyuluIdavalapati So far `diskdict` is Py2 compatible and this pull request is making it required Py3? If yes, we should preserve Py2 compatibility or we risk breaking downstream software depending on this module.

**Comment by prashanthellina** [comment]
Asked @RamanjaneyuluIdavalapati a query above  https://github.com/deep-compute/diskdict/pull/11#discussion_r283851166

File: README.md (hunk 1 above)
Comment (RamanjaneyuluIdavalapati) [reply README.md]: @prashanthellina My bad. It is also compatible for Py2. I don't know why I have changed this. Will give another commit by keeping it as `Python`

**Pull request merged**